starters/http/dotnet/infra/core/host/functions-flexconsumption.bicep

```diff
@@ -11,6 +11,7 @@ param virtualNetworkSubnetId string = ''
 param identityType string
 @description('User assigned identity name')
 param identityId string
+param identityClientId string
```

Comment:
I think you still want to remove this line tho ?
```suggestion
```

Reply:
Good catch